src/libraries/Microsoft.PowerFx.Core/Parser/ParseResult.cs

```diff
@@ -46,6 +46,9 @@ public class ParseResult : IOperationStatus
 
         internal SourceList After { get; }
 
+        // Options used ot create these results.
+        public ParserOptions Options { get; internal set; }
```

Comment:
>Options [](http://example.com/codeflow?start=29&length=7)

ParResult is an out paramaters
Why would in parameters be there?

Reply:
?

Reply:
We call Parse with ParseOptions and this returns a ParseResult.
Why should we include this input param in the output/result ?